tests/data/pipeline-module/meta.tex

```diff
@@ -0,0 +1,3 @@
+% AUTO-GENERATED by Makefile
```

Comment:
Is this file supposed to be in the repo or purely dynamic?

Reply:
Normally it wouldn't be, but I'm persisting it in this so that we don't need to run Docker and LaTeX as part of the test suite.